honeybee_energy/simulation/parameter.py

```diff
@@ -433,6 +433,262 @@ def to_idf(self, identifier='Building'):
+
+                Output:Variable,
+                    *,                        !- key value
+                    Water Heater Electricity Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Water Heater NaturalGas Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Water Use Equipment Heating Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Zone Electric Equipment Electricity Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Zone Gas Equipment NaturalGas Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Zone Ideal Loads Supply Air Total Cooling Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Zone Ideal Loads Supply Air Total Heating Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Zone Lights Electricity Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Zone Other Equipment Lost Heat Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Zone Other Equipment Total Heating Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Zone VRF Air Terminal Cooling Electricity Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Zone VRF Air Terminal Heating Electricity Energy, !- name
+                    Hourly;                   !- frequency
+
+                Output:Variable,
+                    *,                        !- key value
+                    Zone Ventilation Fan Electricity Energy, !- name
+                    Hourly;                   !- frequency
+
```

Comment:
Having all of these output vairables here is making the IDF string a little unread-able. What if you just use this code to make the object before you run to_idf:

```
sim_par = SimulationParameter()
sim_par .output.add_output('Zone Ideal Loads Supply Air Total Heating Energy')
sim_par .output.add_output('Zone Ideal Loads Supply Air Total Cooling Energy')
print(sim_par.to_idf()
```